tests/framework/di/ContainerTest.php

```diff
@@ -286,4 +287,26 @@ public function testContainerSingletons()
         $sameFoo = $container->get('qux.using.closure');
         $this->assertSame($foo, $sameFoo);
     }
+
+    /**
+     * @requires PHP 5.6
```

Comment:
https://phpunit.de/manual/current/en/incomplete-and-skipped-tests.html#incomplete-and-skipped-tests.requires.tables.api